### PR TITLE
Add link to HVS docs from docker scan CLI page

### PR DESCRIPTION
**Pull request by usha-mandya**
Added a link to reference Hub Vulnerability Scan docs from the `docker scan` CLI page.

### CODE REVIEW

**Comment by usha-mandya** [comment]
@mikeparker @mkvitnitsky Could you PTAL?

File: engine/scan/index.md [hunk 1]
```diff
@@ -15,6 +15,8 @@ Vulnerability scanning for Docker local images runs on Snyk engine, providing us
 > Docker vulnerability scanning for local images  is currently a beta release. The commands and flags are subject to change in subsequent releases.
 {:.important}
 
+This page contains information about the `docker scan` CLI command. For information about automatically scanning Docker images through Docker Hub, see [Hub Vulnerability Scanning](/docker-hub/vulnerability-scanning/).
```
Comment (thaJeztah) [inline engine/scan/index.md:18]: Wondering if it should be a note (there's a note about beta feature above, which may distract from this line)

Would it stand out more if it was above the warning?

Comment (usha-mandya) [reply engine/scan/index.md:18]: Yea, I thought about it, but then two consecutive notes may not look good. I can move this above the existing Note though and retain as a separate sentence.

**Review by thaJeztah — APPROVED** [review]
LGTM, thanks!

**Pull request merged**